$OpenBSD: patch-src_fr-command-unarchiver_c,v 1.2 2015/05/01 15:55:41 ajacoutot Exp $

Don't define _XOPEN_SOURCE which would mask the declaration of strptime()

--- src/fr-command-unarchiver.c.orig	Fri Apr  3 13:31:00 2015
+++ src/fr-command-unarchiver.c	Fri Apr  3 13:34:03 2015
@@ -20,7 +20,9 @@
  */
 
 #include <config.h>
+#ifdef __linux__
 #define _XOPEN_SOURCE       /* See feature_test_macros(7) */
+#endif
 #include <time.h>
 #include <stdio.h>
 #include <stdlib.h>
